Content.Server/GameObjects/Components/PlaceableSurfaceComponent.cs

```diff
@@ -19,7 +20,9 @@ public override void ExposeData(ObjectSerializer serializer)
             serializer.DataField(ref _isPlaceable, "IsPlaceable", true);
         }
 
-        public bool InteractUsing(InteractUsingEventArgs eventArgs)
+#pragma warning disable 1998
```

Comment:
So you CAN avoid this warning by not marking the method in question as `async` and then using `Task.FromResult()` to wrap the return value buuuut screw that noise so I'd be open to just doing a project wide muting of this warning.

Reply:
Agreed, I added `<NoWarn>1998</NoWarn>` to the project and removed all added pragmas.